test/java/org/opendatakit/briefcase/ui/export/ExportPanelPageObject.java

```diff
@@ -40,7 +40,7 @@ private ExportPanelPageObject(ExportPanel component, FrameFixture fixture) {
 
   static ExportPanelPageObject setUp(Robot robot) {
     ExportPanel exportPanel = GuiActionRunner.execute(() -> {
-      ExportPanel ep = ExportPanel.from(new TerminationFuture(), new BriefcasePreferences(InMemoryPreferences.empty()));
+      ExportPanel ep = ExportPanel.from(new TerminationFuture(), new BriefcasePreferences(InMemoryPreferences.empty()), new BriefcasePreferences(InMemoryPreferences.empty()), Runnable::run);
```

Comment:
Down in the load method, I think this reads so much better:
```
transferSettings.put(formId, new ServerConnectionInfo(
    getPrefOrThrow(appPreferences, urlKey),
    getPrefOrThrow(appPreferences, usernameKey),
    getPrefOrThrow(appPreferences, passwordKey).toCharArray()
));

  private static String getPrefOrThrow(BriefcasePreferences appPreferences, String key) {
    return appPreferences.nullSafeGet(key).orElseThrow(() -> new RuntimeException("Null value saved for " + key));
  }
```
What do you think?